README.md

```diff
@@ -12,6 +12,22 @@ API](https://developers.google.com/google-ads/api/docs/start).
 
 ## Requirements
 
+*   Two PHP extensions need to be installed to use the library at runtime
```

Comment:
I don't think we need to put this as a first line. It can be grouped under the bullet you've just removed "System requirements and dependencies..."

Reply:
Done